release-schema.json

```
@@ -14,7 +14,45 @@
         "legalBasis": {
           "title": "Legal basis",
           "description": "The legal basis of the procedure.",
```

The existing description simply restated the title.

```suggestion
          "description": "The laws and regulations that govern the contracting process and grant legal authority to the procuring entity.",
```